prime-dotnet-webapi/ViewModels/AutoMapping.cs

```diff
@@ -80,5 +80,9 @@ public AutoMapping()
         CreateMap<AuthorizedUser, AuthorizedUserViewModel>()
             .IncludeMembers(src => src.Party);
         CreateMap<Party, AuthorizedUserViewModel>();
+
+        CreateMap<AgreementVersion, AgreementVersionViewModel>()
+            .ForMember(dest => dest.UpdatedDate, opt => opt.MapFrom(src => src.UpdatedTimeStamp))
+            .ForMember(dest => dest.AgreementContent, opt => opt.MapFrom(src => src.Text));
```

Comment:
I'm not sure why these properties are named differently in the DB vs the ViewModel? 
`Agreements` have a property called AgreementContent, while `AgreementVersion`s have Text. The main difference is that AgreementContent is also rendered with Razor and includes styling, additional clauses, and could include other variables such as today's date or the person's name, etc. To me, it seems like keeping the VM property Text instead of AgreementContent is more accurate?

Reply:
For reference, we want to generally avoid using the audit columns whenever possible, since they are more like an implementation detail than actual business data. We could, as a crazy hypothetical example, have a database trigger that pushes a log whenever we update a row. Or implement temporal tables. In this case however, it's fine for now at least since its what they want in the existing mock-ups. I feel we should probably still make another column called PublishedDate or something like that rather than using the audit columns

Reply:
Also for reference, once published we won't be updating the AgreementVersions, we will only be inserting new records. These represent records of legal agreements, so we cant have the admins go back and ret-con an agreement someone already signed.

Reply:
> For reference, we want to generally avoid using the audit columns whenever possible, since they are more like an implementation detail than actual business data. We could, as a crazy hypothetical example, have a database trigger that pushes a log whenever we update a row. Or implement temporal tables. In this case however, it's fine for now at least since its what they want in the existing mock-ups. I feel we should probably still make another column called PublishedDate or something like that rather than using the audit columns

Totally agree with the db trigger implementation approach for audit date (or any other columns possible) its less error prone so developer won't forget assigning value to them. I think right now it's okay since the ticket is just previewing and hopefully more implementation will be added in future ticket.